resource-managers/kubernetes/docker-minimal-bundle/src/main/docker/executor/Dockerfile

```diff
@@ -29,4 +29,5 @@ CMD SPARK_CLASSPATH="${SPARK_HOME}/jars/*" && \
     if ! [ -z ${SPARK_EXECUTOR_EXTRA_CLASSPATH+x} ]; then SPARK_CLASSPATH="$SPARK_EXECUTOR_EXTRA_CLASSPATH:$SPARK_CLASSPATH"; fi && \
     if ! [ -z ${SPARK_EXTRA_CLASSPATH+x} ]; then SPARK_CLASSPATH="$SPARK_EXTRA_CLASSPATH:$SPARK_CLASSPATH"; fi && \
     if ! [ -z ${SPARK_MOUNTED_FILES_DIR} ]; then cp -R "$SPARK_MOUNTED_FILES_DIR/." .; fi && \
+    if ! [ -z ${SPARK_MOUNTED_FILES_FROM_SECRET_DIR} ]; then cp -R "$SPARK_MOUNTED_FILES_FROM_SECRET_DIR/." .; fi && \
```

Comment:
Some other options:

1) Make the working directory for the base image be somewhere other than the SPARK_HOME directory, such that it's guaranteed to just be an empty directory and thus adding contents to it won't overwrite anything.
2) Make the files in SPARK_HOME immutable
3) Have a script do the copying such that it intelligently will throw an error if it attempts to overwrite something

Of these three options, I like (1) the most, though (3) is acceptable. (2) is very difficult to achieve.